package.json

```diff
@@ -15,8 +15,12 @@
         "dist"
     ],
     "scripts": {
-        "build": "webpack --mode production",
-        "watch": "webpack --watch --mode development",
+        "build": "npm-run-all -s \"clean-dist\" \"create-version\" \"build:web-sdk\"",
+        "build:web-sdk": "webpack --mode production",
+        "watch": "npm-run-all -s \"clean-dist\" \"create-version\" \"watch:web-sdk\"",
+        "watch:web-sdk": "webpack --watch --mode production",
```

Comment:
```suggestion
        "build-sdk": "npm-run-all -s \"clean-dist\" \"create-version\" \"build\"",
        "build": "webpack --mode production",
        "watch-sdk": "npm-run-all -s \"clean-dist\" \"create-version\" \"watch\"",
        "watch": "webpack --watch --mode development",
```

Reply:
> 
> 
>     * The README file needs updating: https://github.com/vircadia/vircadia-web-sdk/blob/master/README.md
> 
>     * The version number needs to be removed from or set to 0.0.0 in package.json so that it isn't confusing.
> 
>     * The copyStatic.js file isn't needed.

Isn't the version necessary though in package.json? This script could perhaps update the package.json in the same way NPM updates package-lock every time NPM does something.

Reply:
Yes, the version number in the package.json probably is necessary, which subsequently occurred to me. (I assume that the version number is read from package.json.)